BREAKING.md

```diff
@@ -21,8 +21,8 @@ fortunately migration tools are provided to ease the task.
 * The configuration mostly remained the same, only one major key has been added: `jwt_secret`
 and one key removed: `secure` from the SMTP notifier as the Go SMTP library default to TLS
 if available.
-* The Hash router has been replaced by a Browser router. This means that the weird characters
-/%23/ in the URL could now be safely removed.
+* The Hash router has been removed and replaced with a Browser router. This means that the weird characters
```

Comment:
This is already merged in @SilverBut 's PR, you should probably rebase.

Reply:
I merged the PR into this branch and not master, hence why you're seeing it here.
I actually rebased it into this PR.